### DISCUSSION TITLE
ERROR - Failed to load the extension "coqui_tts"

### DISCUSSION THREAD

**Discussion by zuzirtex** [oobabooga/text-generation-webui#6075] [Q&A]
Ao tentar aplicar a extensão "coqui_tts", aparece a seguinte mensagem de erro no terminal CMD: **ERROR   Failed to load the extension "coqui_tts"**.

Abri o "cmd_windows.bat" e rodei o comando **pip install -r extensions\coqui_tts\requirements.txt --upgrade**, porém, não resolveu. O que posso fazer para resolver isso?

![image](https://github.com/oobabooga/text-generation-webui/assets/96734138/2dfc2243-e4e7-45ac-9271-a2656e27a04b)

![image](https://github.com/oobabooga/text-generation-webui/assets/96734138/797b1e05-2195-474b-9fa3-f9e8f869fcf1)

**Comment by blackmambaza** [comment]
fazem isto:

- executar cmd_windows.bat

- então vá para \installer_files\env\bin\python.exe -m pip install TTS

- reinicie

**Comment by zuzirtex** [comment]
Tentei seguir esse passo, mas ocorrem os seguintes erros:

![image](https://github.com/oobabooga/text-generation-webui/assets/96734138/494d0aff-bc82-4086-8258-6871373672a3)
![image](https://github.com/oobabooga/text-generation-webui/assets/96734138/9fb1a38a-f2b5-4b1b-ae41-6ed26df8ceff)

**Reply by blackmambaza** [reply]
Tente fazer o downgrade para python 3.9

**Comment by israelsouzaart** [comment]
Testei também, não funcionou, acredito que alguma opção tenha sido modificada, pois não está funcionando mais.

**Comment by bolder004** [comment]
I have the same problem with no solution.

**Reply by blackmambaza** [reply]
Follow the same steps I initially commented with

**Comment by israelsouzaart** [comment]
Meu para Windows, nada do que ensinou fez sentido no Windows.

Em ter., 15 de out. de 2024 às 13:05, blackmambaza ***@***.***>
escreveu:
